<?xml version="1.0"?>
<launch>
    <param name="/use_sim_time" value="false" />

    <node name="map_server" pkg="map_server" type="map_server" args="$(find navigation)/map/505_1.yaml" >
        <param name="frame_id" value="map" />
    </node>
    <!-- <node name="costmap_clean" pkg="navigation" type="costmap_clean"/> -->
    <node pkg="tf" type="static_transform_publisher" name="odom2camera_init" args="0 0 0 0 0 0 odom camera_init 1" />
    <node pkg="tf" type="static_transform_publisher" name="baselink2body" args="-0.068 0 -0.365 0 -0.3491 0 body base_link  1" />
    <node pkg="tf" type="static_transform_publisher" name="vicemap2map"  args="0.189 0 0.320 0 0.3491 0 map vice_map 1" />
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" >
        <rosparam file="$(find navigation)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find navigation)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find navigation)/config/move_base_params.yaml" command="load" />
        <rosparam file="$(find navigation)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find navigation)/config/global_costmap_params.yaml" command="load" />

        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <param name="neutral_cost" value="220" />
        <param name="planner_frequency" value="3.0" />
        <param name="planner_patience" value="1.0" />
        <param name="use_dijkstra" value="false" />
        

        <param name="clearing_rotation_allowed" value="false" />

        <remap from = "/cmd_vel" to ="/"/>
    </node>


</launch>